scripts/frame-weight-template.hbs

```diff
@@ -0,0 +1,86 @@
+//! Autogenerated weights for {{pallet}}
```

Comment:
Why do we need this? I removed it a few commits ago in favor of the default substrate one, and it works fine for the Centrifuge runtime.

Reply:
We need a version of the template that will generate the actual trait. The default one only generates an impl

Reply:
Ok, will test it out when generating the runtime ones.